docs/index.rst

```diff
@@ -8,14 +8,16 @@ hosting RPM family content types.
 Features
 --------
 
-* :ref:`sync-publish-workflow` with "RPM Content" including RPMs and Errata
+* :ref:`sync-publish-workflow` with "RPM Content" including RPMs, Errata, Modularity, and Comps
```

Comment:
Pavel is working on another docs PR that moves all errata-->advisory, so i suggest sticking to advisory term in this PR as well.